packages/vue-language-core/src/plugins/file-md.ts

```diff
@@ -9,6 +9,7 @@ const scriptSetupReg = /\\\<[\s\S]+?\>\n?/g;
 const sfcBlockReg = /\<(script|style)\b[\s\S]*?\>([\s\S]*?)\<\/\1\>/g;
 const angleBracketReg = /\<\S*\:\S*\>/g;
 const linkReg = /\[[\s\S]*?\]\([\s\S]*?\)/g;
+const codeSnippetImportReg = /^<<<\s*.+/gm;
```

Comment:
`<<<` can be indented too: https://github.com/brc-dd/vitepress-whyframe-demo/blob/main/src/components/alert/index.md?plain=1#L21

Reply:
Makes sense, thanks!